app/views/pages/roadmap.liquid

```diff
@@ -11,11 +11,23 @@ metadata:
 
 This section contains information about features we are implementing or planning to implement.
 
-## 2019
+## H2 2019
```

Comment:
?

Reply:
Second half of 2019

Reply:
Can we write "Second half of 2019" instead of this? Im pretty sure there is 0 benefit in hiding this behind shortcut that can mean a lot of different things, including [hydrogen](https://pubchem.ncbi.nlm.nih.gov/compound/Hydrogen), [heading 2](https://www.quackit.com/html/tags/html_h2_tag.cfm) and a [car model](https://en.wikipedia.org/wiki/Hummer_H2) ;)

Reply:
I didn't know about the car model :) Edited based on your suggestion. Thank you!